### ISSUE TITLE
Issues when migrating to a new phase with fact sheet answers/different tasks that outputs identical data

### ISSUE THREAD

**Issue by liviust** [codalab/codabench#1472]
Let **COMP** be a competition with 2 phases named _DEVELOP_ and _TEST,_ each with identical tasks. Set the results to '**Force Latest Multiple**' and add a few **mandatory Fact Sheet** entries. Submit several runs with Fact Sheet responses for the _TEST_ phase.

Now, add _MIGRATE_ as a new phase with a new task created through the Resources menu, **containing the exact same files** (scoring scripts, etc.) as the task created for the _TEST_ phase. During the migration of submissions from _TEST_ to _MIGRATE_, the leaderboard displays only 'n/a' values. However, upon re-evaluation of the runs in the migration process, the actual submissions do have values, visible in the submission list of the _MIGRATION_ phase task. The only difference is that in the **ADMIN** console, the _Fact Sheet_ answer list is null.

Now, edit the **COMP** settings, and replace the _MIGRATE_ phase task with the _TEST_ phase task. the Fact Sheet answers are still n/a but now, the values are displayed correctly in the Leaderboard.

Remove the Fact Sheet questions from the **COMP** settings; now the results are displayed correctly.

In conclusion, there is an issue with: 1) the Fact Sheet answers when migrating the submissions, 2) the submissions results when using other tasks for the new phase that output the same exact data as the task from the previous phase. The answers are not migrated, and the actual results are displayed with n/a.

**Comment by ihsaan-ullah** [comment]
> Now, edit the **COMP** settings, and replace the _MIGRATE_ phase task with the _TEST_ phase task. the Fact Sheet answers are still n/a but now, the values are displayed correctly in the Leaderboard.


I am afraid that the submissions in ***MIGRATE*** phase are run on ***TEST*** Phase task. That is also the reason for seeing `n/a` on the leaderboard. I don't understand completely the issue with factsheet answers but will check this in detail soon.

**Comment by Didayolo** [comment]
I close this issue to merge and keep track of it here: #1540.
